packages/matrix/tests/login.spec.ts

```diff
@@ -73,6 +79,31 @@ test.describe('Login', () => {
     await expect(page.locator('[data-test-login-form]')).toBeVisible();
   });
 
+  // TODO: figure out why this test why updateDisplayName is not triggering the event in matrix-service
+  test.skip('the profile reflects display name changes', async ({
+    page,
+    synapse,
+  }) => {
+    page.on('console', (msg) => console.log(msg.text()));
+    await login(page, 'user1', 'pass');
+
+    await page.locator('[data-test-profile-icon-button]').click();
+    await expect(page.locator('[data-test-profile-display-name]')).toHaveText(
+      'user1',
+    );
+
+    await updateDisplayName(
+      synapse,
+      registeredUser!.userId,
+      registeredUser!.accessToken,
+      'newname',
+    );
+
+    await expect(page.locator('[data-test-profile-display-name]')).toHaveText(
+      'newname',
+    );
+  });
+
```

Comment:
i think we can remove this test--seems like an unlikely scenario given the current requirements